src/MCRun.jl

```diff
@@ -74,15 +74,20 @@ function MCState(temp,beta, config::Config, pot::AbstractMLPotential;kwargs...)
     en_tot = RuNNer.getenergy(pot.dir, config,pot.atomtype)
 
     MCState(temp, beta, config, dist2_mat, en_atom_vec, en_tot; kwargs...)
-end 
-function MCState(temp,beta, config::Config, pot::DFTPotential;kwargs...)
+
```

Comment:
what happened to the method for DFTPotential?

Reply:
reinserted